src/doc/rustc/src/codegen-options/index.md

```diff
@@ -403,6 +403,15 @@ linkage. This flag takes one of the following values:
 * `y`, `yes`, `on`, or no value: use dynamic linking.
 * `n`, `no`, or `off`: use static linking (the default).
 
+## print-link-args
+
+This flag causes `rustc` to print the full linker invocation. This can be
+useful when debugging linker options.
+
+The exact format of this debugging output is not a stable guarantee, other than
+that it will include the text of each command-line argument passed to the
+linker.
```

Comment:
Just a small suggestion: it might be good to call out that this will also print the path & executable name of the linker. On Windows, we have a set of search heuristics to find the correct linker to use and don't just rely on the linker being on the path. If someone is trying to figure out *which* linker is being run, this would also be the appropriate flag to set.